custom_model_runner/datarobot_drum/drum/language_predictors/java_predictor/java_predictor.py

```diff
@@ -21,6 +22,7 @@
 class JavaPredictor(BaseLanguagePredictor):
     JAVA_COMPONENT_ENTRY_POINT_CLASS = "com.datarobot.custom.PredictorEntryPoint"
     JAVA_COMPONENT_CLASS_NAME = "com.datarobot.custom.ScoringCode"
```

Comment:
Please change var name to : JAVA_COMPONENT_CLASS_NAME_DATAROBOT

Reply:
done
`class JavaPredictor(BaseLanguagePredictor):
    JAVA_COMPONENT_ENTRY_POINT_CLASS = "com.datarobot.custom.PredictorEntryPoint"
    JAVA_COMPONENT_CLASS_NAME_DATAROBOT = "com.datarobot.custom.ScoringCode"
    JAVA_COMPONENT_CLASS_NAME_H2O = "com.datarobot.custom.H2OPredictor"`